src/visitors/visitor_utils.cpp

```diff
@@ -260,4 +260,19 @@ std::pair<std::string, std::unordered_set<std::string>> statement_dependencies(
     return {key, out};
 }
 
+std::string get_indexed_name(const ast::IndexedName& node) {
+    return node.get_node_name() + "[" + to_nmodl(node.get_length()) + "]";
+}
+
+std::string get_full_var_name(const ast::VarName& node) {
```

Comment:
also this could be a method of the class `VarName`

Reply:
For reasons applied at the above function I think it's fine to keep it as it is